Tests/SwiftLayoutTests/PerformanceTests.swift

```diff
@@ -40,15 +45,30 @@ class PerformanceTests: XCTestCase {
         }
     }
     
+    func testPerformanceNativeApi() throws {
+        let metrics: [XCTMetric] = [XCTCPUMetric(), XCTMemoryMetric()]
+        self.measure(metrics: metrics) {
+            _ = NativeApiView(frame: .init(x: 0, y: 0, width: 375, height: 667))
+        }
+    }
+    
     func testPerformanceLayoutable() throws {
         let metrics: [XCTMetric] = [XCTCPUMetric(), XCTMemoryMetric()]
-
         self.measure(metrics: metrics) {
             _ = SwiftLayoutView(frame: .init(x: 0, y: 0, width: 375, height: 667))
         }
     }
     
     func testPerformanceInterfaceBuilderAndLayout() throws {
+        let metrics: [XCTMetric] = [XCTCPUMetric(), XCTMemoryMetric()]
```

Comment:
mismatching interface and native 😉

Reply:
haha! Oh my, did I make this mistake? 😅
I corrected it.